tests/trestle/utils/load_distributed_test.py

```diff
@@ -0,0 +1,138 @@
+# -*- mode:python; coding:utf-8 -*-
+
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for trestle load_distributed module."""
+
+import shutil
+from pathlib import Path
+
+import dictdiffer
+
+from tests import test_utils
+
+from trestle.oscal.catalog import Catalog, ResponsibleParty, Role
+from trestle.utils import fs
+from trestle.utils.load_distributed import load_dict, load_distributed, load_list
+
+
+def test_load_list(testdata_dir, tmp_trestle_dir):
+    """Test loading a list recursively."""
+    # prepare trestle project dir with the file
+    test_utils.ensure_trestle_config_dir(tmp_trestle_dir)
```

Comment:
Do we really need to do this? tmp_trestle_dir will fail if it is not created. We probably should kill that function.

Reply:
Need a initialised trestle directory to copy the testdata. Testdata does not include a initialised trestle project.